fdbserver/DDRelocationQueue.actor.cpp

```diff
@@ -1252,6 +1252,18 @@ void traceRelocateDecision(TraceEvent& ev, const UID& pairId, const RelocateDeci
 		ev.detail("ShardSize", decision.metrics.bytes).detail("ParentShardSize", decision.parentMetrics.get().bytes);
 	}
 }
+
+int nonOverlappedSrcCount(const std::vector<UID>& srcIds, const std::vector<UID>& destIds) {
```

Comment:
[Optional] It might be clearer to use `Server` instead of `Src` in the function name since we're comparing the source servers to destination servers. (So the function could be something like `nonOverlappingServerCount`.)

Reply:
copy that